data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/cloudwatch_logs/config/ThresholdConfig.java

```diff
@@ -22,23 +23,27 @@ public class ThresholdConfig {
     public static final int DEFAULT_BATCH_SIZE = 25;
     public static final String DEFAULT_EVENT_SIZE = "256kb";
     public static  final String DEFAULT_SIZE_OF_REQUEST = "1mb";
-    public static final long DEFAULT_LOG_SEND_INTERVAL_TIME = 60;
+    public static final long DEFAULT_FLUSH_INTERVAL = 60;
 
-    @JsonProperty("batch_size")
-    @Size(min = 1, max = 10000, message = "batch_size amount should be between 1 to 10000")
+    @JsonProperty(value = "batch_size", defaultValue="25")
+    @Min(1)
```

Comment:
@Min and @Max are only applicable to numeric types. For string @Size remains correct